tests/nested/manual/uc-update-assets-secure/task.yaml

```diff
@@ -50,9 +55,12 @@ prepare: |
   cp shim.efi.out pc/shim.efi.signed
   tests.nested secboot-sign file pc/shim.efi.signed "${SNAKEOIL_KEY}" "${SNAKEOIL_CERT}"
 
+  tests.nested secboot-remove-signature pc/grubx64.efi
   if [ "${UPDATE_SEED}" = true ]; then
     # Resign grub with new vendor key
     tests.nested secboot-sign file pc/grubx64.efi "new-cert.key" "new-cert.crt"
+    # We need to keep the old vendor key signature to allow hard reset before new shim is installed
```

Comment:
secboot does not like things that do not make sense. Because this case has to work, we have to give the bootchain to secboot. It will fail if it cannot figure out how signatures are verified.